### ISSUE TITLE
Compile error on install

### ISSUE THREAD

**Issue by margaretjoanmiller** [pjmp/sunny#1]
Hi. I'm trying to "cargo install" this package and I get this error:
```
error[E0432]: unresolved import `clap::Clap`
 --> /home/maeday/.cargo/registry/src/github.com-1ecc6299db9ec823/sunny-0.2.0/src/cli.rs:3:5
  |
3 | use clap::Clap;
  |     ^^^^^^^^^^ no `Clap` in the root

error: cannot determine resolution for the derive macro `Clap`
 --> /home/maeday/.cargo/registry/src/github.com-1ecc6299db9ec823/sunny-0.2.0/src/cli.rs:9:17
  |
9 | #[derive(Debug, Clap)]
  |                 ^^^^
  |
  = note: import resolution is stuck, try simplifying macro imports

error: cannot find attribute `clap` in this scope
  --> /home/maeday/.cargo/registry/src/github.com-1ecc6299db9ec823/sunny-0.2.0/src/cli.rs:10:3
   |
10 | #[clap(about, version, after_help = "note: run --help to see more details")]
   |   ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> /home/maeday/.cargo/registry/src/github.com-1ecc6299db9ec823/sunny-0.2.0/src/cli.rs:13:7
   |
13 |     #[clap(display_order = 1, parse(from_str = from_str))]
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> /home/maeday/.cargo/registry/src/github.com-1ecc6299db9ec823/sunny-0.2.0/src/cli.rs:17:7
   |
17 |     #[clap(short, long, display_order = 2, validator = validate_path, parse(from_str = expand_tilde), long_about = r"
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> /home/maeday/.cargo/registry/src/github.com-1ecc6299db9ec823/sunny-0.2.0/src/cli.rs:24:7
   |
24 |     #[clap(long)]
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
  --> /home/maeday/.cargo/registry/src/github.com-1ecc6299db9ec823/sunny-0.2.0/src/cli.rs:28:7
   |
28 |     #[clap(
   |       ^^^^
   |
   = note: `clap` is in scope, but it is a crate, not an attribute

error[E0599]: no function or associated item named `parse` found for struct `cli::Config` in the current scope
  --> /home/maeday/.cargo/registry/src/github.com-1ecc6299db9ec823/sunny-0.2.0/src/cli.rs:55:17
   |
11 | pub struct Config {
   | ----------------- function or associated item `parse` not found for this
...
55 |         Config::parse()
   |                 ^^^^^ function or associated item not found in `cli::Config`
   |
   = help: items from traits can only be used if the trait is implemented and in scope
   = note: the following traits define an item `parse`, perhaps you need to implement one of them:
           candidate #1: `Parser`
           candidate #2: `time::private::Parsable`

Some errors have detailed explanations: E0432, E0599.
```

**Comment by pombadev** [comment]
Not sure what's going on but can you try installing with `--locked` flag?

```bash
cargo install sunny --locked
```

**Comment by pombadev** [comment]
fixed in v0.2.3, please try `cargo install sunny` again. If it's still an issue, please reopen this ticket, thanks!

**Comment by margaretjoanmiller** [comment]
That worked!
